public/data/sameem420.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Muhammad Sameem",
+  "type": "personal",
+  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

Comment:
```suggestion
  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

Reply:
I think the previous one is totally fine because the bio is really long already 😅

Reply:
There's double `and` (& and)! That's why I asked to remove one AND 😄